lua/harpoon/list.lua

```diff
@@ -285,6 +285,17 @@ function HarpoonList:resolve_displayed(displayed, length)
     end
 end
 
+function HarpoonList:sync_cursor()
+    local current_display = Utils.normalize_path(
```

Comment:
Calling this `current_display` is a bit misleading as it is used in `self:get_by_value(current_display)`.

Reply:
Any sugs for what we might want to name for this variable instead? What do you think about  `displayed_path`?

Reply:
I would go with `current_path`.